src/include/osl_pvt.h

```diff
@@ -629,6 +629,18 @@ class Symbol {
         m_lastwrite  = std::max (m_lastwrite, lw);
     }
 
+    // Mark the symbol as always being read (and, if write==true, also
+    // that it's always written). Then is for when we don't know when
+    // it's read or written, but want to be sure it doesn't look unused.
+    void mark_always_used (bool write=false) {
```

Comment:
`write` stays false in all the uses so far?

So the newly added symbols always only read from, never written to?